packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -1231,6 +1231,38 @@ void main() {
     expect(Focus.of(firstChildKey.currentContext!).hasPrimaryFocus, isFalse);
     expect(Focus.of(secondChildKey.currentContext!).hasPrimaryFocus, isTrue);
   });
+
+  testWidgets('CheckboxListTile use ListTileTheme controlAffinity', (WidgetTester tester) async {
+    Widget buildView(ListTileControlAffinity controlAffinity) {
+      return MaterialApp(
+        home: Builder(builder: (BuildContext context) {
+          return Theme(
+            data: ThemeData.light().copyWith(
+              listTileTheme: ListTileTheme.of(context).copyWith(
+                controlAffinity: controlAffinity,
+              ),
+            ),
+            child: Material(
+              child: CheckboxListTile(
+                value: false,
+                onChanged: (bool? value) {},
+              ),
+            ),
+          );
+        }),
+      );
+    }
```

Comment:
```suggestion
    Widget buildListTile(ListTileControlAffinity controlAffinity) {
      return MaterialApp(
        theme: ThemeData(
          listTileTheme: ListTileThemeData(
            controlAffinity: controlAffinity,
          ),
        ),
        home: Material(
          child: CheckboxListTile(
            value: false,
            onChanged: (bool? value) {},
          ),
        ),
      );
    }
```

Please update this structure here and similar in other test classes